subprojects/gradle/cd/src/main/kotlin/com/avito/ci/steps/UploadToQapps.kt

```diff
@@ -19,13 +22,19 @@ class UploadToQapps(context: String, artifactsConfiguration: ArtifactsConfigurat
             "${this.javaClass.simpleName} check requires $plugin being applied to project"
         }
 
+        val uploadCondition = UploadToQappsCondition(project.cdBuildConfig.orNull)
+
         return artifactsMap.map { (_, output) ->
             when (output) {
                 is Output.ApkOutput ->
                     project.tasks.qappsTaskProvider(output.variantName)
                         .apply {
                             configure {
                                 it.getApk().set(File(output.path))
+                                it.releaseChain.set(uploadCondition.isReleaseChain())
+                                it.onlyIf {
+                                    uploadCondition.canUpload()
```

Comment:
Do we send PR artifacts to qapps?

Reply:
No, at least for now. 
There is no conflict in the requirements. Cd build config can only disable uploading.